content/docs/manifest.json

```diff
@@ -39,10 +39,6 @@
               "title": "OperatorHub (OLM)",
               "path": "/docs/installation/operator-lifecycle-manager.md"
             },
-            {
-              "title": "Other tools",
-              "path": "/docs/installation/other-tools.md"
-            },
```

Comment:
suggestion: Rather than entirely removing this page, could we maybe replace the kubeprod installation instructions with a notice saying that it's no longer supported?

If we remove this page it'll break any links that people have to the page and we try to avoid that!

(another approach is to add a redirect from the old older-tools page to a different page, but that might be confusing!)

Reply:
I think it is clear that the kube-prod-runtime project is archived & should not be used anymore.
I think it is good to simplify the side menu and remove this item.
For that reason, I would prefer just adding a redirect.